src/internal.c

```diff
@@ -15719,8 +15719,22 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
 
                             /* do not add to certificate manager */
                             skipAddCA = 1;
+                        } /* ASN_NO_SIGNER_E || ASN_SELF_SIGNED_E */
+                    } /* ret != 0 && isCA */
+                #else
+                	/* When WOLFSSL_DEBUG_CERTS enabled, suggest solution */
```

Comment:
```weird control chars, hard tabs, CRs, trailing whitespace:
./src/internal.c:15725:                »/* When WOLFSSL_DEBUG_CERTS enabled, suggest solution */```

Reply:
Ugh. _Usually_ VS converts tabs to spaces automatically. Fixed.